server: some tg assets prs

ROdenFL opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
https://github.com/tgstation/tgstation/pull/63503
https://github.com/tgstation/tgstation/pull/64579
https://github.com/tgstation/tgstation/pull/70753
https://github.com/tgstation/tgstation/pull/74884
https://github.com/tgstation/tgstation/pull/80522
https://github.com/tgstation/tgstation/pull/69852
https://github.com/tgstation/tgstation/pull/80601

<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений
<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->

ROdenFL committed:
server: some tg assets prs

Bizzonium commented on code/modules/asset_cache/assets/asset_emoji.dm:
```diff
@@ -1,5 +1,5 @@
-/datum/asset/spritesheet/emoji/create_spritesheets()
-	//InsertAll("emoji", EMOJI_SET)
+// /datum/asset/spritesheet/emoji/create_spritesheets()
+// 	//InsertAll("emoji", EMOJI_SET)
```
А почему закомменчено?

ROdenFL replied:
у нас нету их

ROdenFL replied:
как будут раскоменчу

ROdenFL committed:
Safe

Bizzonium committed:
Merge branch 'master220' into assetcache

Bizzonium committed:
Sync more with TG

https://github.com/tgstation/tgstation/pull/69852
https://github.com/tgstation/tgstation/pull/80601
